docs/docs/config-colors.md

```diff
@@ -51,5 +51,15 @@ To change *only* the background color, just omit the first color from the above
 "prefix": "<,#FFFFFF>┏[</>",
 ```
 
+## Palette reference
+
+If your theme has a defined [Palette][palette], you can use the Palette reference in places where the
+__Standard color__ is expected.
+
+Palette references have 2 possible formats: `p:<palette key>` and `palette:<palette key>`.
+
+Read more about [Palette][palette].
```

Comment:
I would include palette here, rather than having a separate section.

Reply:
Makes sense, I will change the docs accordingly.

Reply:
Moved palette docs to Palette section in config-colors.md.